x-pack/test/fleet_api_integration/apis/agents/upgrade.ts

```diff
@@ -710,39 +710,6 @@ export default function (providerContext: FtrProviderContext) {
           })
           .expect(400);
       });
-      it('should respond 200 if trying to bulk upgrade to a version higher than the latest fleet server version if the agents include fleet server agents', async () => {
```

Comment:
should there be a new integration test added to cover the new agent version check?